mne/viz/ica.py

```diff
@@ -329,17 +335,42 @@ def plot_ica_properties(ica, inst, picks=None, axes=None, dB=True,
 
     # calculations
     # ------------
+    
     if isinstance(inst, BaseRaw):
+        # if reject is set, reject
+        if reject == 'auto':
+            reject = getattr(ica, 'reject_', None)
+        if reject is not None:
+            data = inst.get_data(ica.picks_)
+            data, drop_inds = _reject_data_segments(data, reject, flat=None, 
+                                                    decim=None, info=ica.info,
+                                                    tstep=2.0)
+            ch_names = np.array(inst.info['ch_names'])
+            ch_names = ch_names[ica.picks_].tolist()
```

Comment:
Style: better do ```ch_names = [inst.info['ch_names'][kk] for kk in ica.picks_]```
Numpy is not needed here and like this you give hints on types.